src/BaGet.Aws/BaGet.Aws.csproj

```diff
@@ -1,16 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-
+    <TargetFramework>net6.0</TargetFramework>
     <PackageTags>NuGet;Amazon;Cloud</PackageTags>
     <Description>The libraries to host BaGet on AWS.</Description>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.S3" Version="3.3.110.20" />
-    <PackageReference Include="AWSSDK.SecurityToken" Version="3.3.104.27" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(MicrosoftExtensionsPackageVersion)" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.7.11" />
+    <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.1.105" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0" />
```

Comment:
Is there a reason you removed the MSBuild variables to manage versions? Using variables helps keep versions consistent across multiple projects. Let me know if you ran into problems though!

Reply:
Uhm, yes, I remember the reason of this change, but now I'm looking into the master branch and think this was done by a mistake. 

Should I revert this variable only or all? (cause it seemed to me that some variables were used only in one project)

Reply:
I think I will revert all deleted variables. Sorry for this

Reply:
**Reverted**